scm/driver/gitea/repo.go

```diff
@@ -51,6 +51,11 @@ func (s *repositoryService) ListV2(ctx context.Context, opts scm.RepoListOptions
 	return s.List(ctx, opts.ListOptions)
 }
 
+func (s *repositoryService) ListNamespace(ctx context.Context, _ string, opts scm.ListOptions) ([]*scm.Repository, *scm.Response, error) {
+	// gitea does not have namespace
+	return s.List(ctx, opts)
```

Comment:
gitea has organizations just like github, no?

Reply:
If there's no API then maybe we should return an error instead of returning wrong data?

Reply:
fixed